src/test/java/io/github/christophermanahan/carnitas/HTTPServerTest.java

```diff
@@ -7,73 +7,71 @@
 import java.util.List;
 import java.util.Optional;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-class EchoServerTest {
+class HTTPServerTest {
 
-  private List<String> received;
-  private List<String> sent;
+  private String received;
+  private List<byte[]> sent;
```

Comment:
Is capturing this in a List of byte arrays that easiest way to do this? Could you just store a list of strings?